bricks/containers/src/resizable-box/index.spec.tsx

```
@@ -0,0 +1,141 @@
+
+jest.mock("@next-core/theme", () => ({}));
+
+jest
+  .spyOn(document.documentElement, "clientWidth", "get")
+  .mockImplementation(() => 800);
+jest
+  .spyOn(document.documentElement, "clientHeight", "get")
+  .mockImplementation(() => 600);
+
+window.requestAnimationFrame = jest.fn(
+  (fn: Function) => setTimeout(fn, 17) as unknown as number
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)